libs/components/toast/src/lib/modules/toast/toast.component.scss

```diff
@@ -156,58 +255,11 @@ sky-toast {
 
     .sky-toast-icon-theme-modern {
       display: block;
-      margin: -5px 0 0 0;
-    }
-
-    .sky-box {
-      &.sky-toast-info {
```

Comment:
I updated our local function for these to allow for setting the icon color and then moved the pieces into the main declarations for these types to reduce code.